src/main/webapp/js/instructor.es6

```diff
@@ -369,6 +369,25 @@ function attachEventToDeleteStudentLink() {
     });
 }
 
+function remindButtonAjax(urlLink) {
```

Comment:
Yes, that's more in line with how we name functions/methods. But I don't have enough knowledge about this code to confirm if the proposed name describes the function's purpose accurately. May be @chowyb can weigh in?